packages/core-components/src/lib/unsorted/viz/Column.svelte

```diff
@@ -66,7 +66,8 @@
 	export let deltaSymbol = true;
 	deltaSymbol = deltaSymbol === 'true' || deltaSymbol === true;
 
-	let options = {
+	let options;
```

Comment:
You don't actually _need_ this declaration; Svelte adds it automatically for you based on the line below

Reply:
Very cool, I didn't know that. Removed, thanks